spec/es5.d.ts

```diff
@@ -0,0 +1,13 @@
+interface Symbol { }
+interface SymbolConstructor {
+  iterator: symbol;
+}
+declare var Symbol: SymbolConstructor;
+interface Iterable<T> {
+    [Symbol.iterator](): Iterator<T>;
+}
+interface Iterator<T> {
```

Comment:
what's difference this to what we're currently using in `es6-shim` type definitions? looks identical.